test/migration.js

```diff
@@ -61,24 +61,27 @@ test('migrate moves msgs from old log to new log', (t) => {
     fromEvent('ssb:db2:migrate:progress', sbot),
     pull.filter((x) => x === 1),
     pull.take(1),
-    pull.drain(() => {
-      // we need to make sure async-flumelog has written the data
-      sbot.db.onDrain(() => {
+    // we need to make sure async-flumelog has written the data
+    pull.asyncMap((_, continueCb) =>
+      setTimeout(() => {
```

Comment:
This setTimeout is just waiting for the fs to be written, seems unrelated to onDrain

Reply:
can't you do a sbot.db.log.onDrain? If you just care about when stuff has been written, that seems like a safer bet

Reply:
So, I wrote this setTimeout hack **before** I decided to export `log` at `sbot.db.log`. Now that we have it, it makes more sense to use `sbot.db.log.onDrain` indeed.